packages/docs/src/styles/components/footer.scss

```diff
@@ -1,2 +1,86 @@
 .c-footer {
+  .ds-footer__container {
+    margin-inline: auto;
+    max-width: $site-max-width;
+    width: 100%;
+  }
+}
+
+.c-footer__content--top,
+.c-footer__content--bottom {
```

Comment:
As a continuation of my comments in the React component, consider scoping these styles to a `c-footer__content` class. This would be more inline with BEM naming